lib/opencensus/trace/integrations/rails.rb

```diff
@@ -75,17 +108,35 @@ class Rails < ::Rails::Railtie
             rc.add_option! :events, DEFAULT_NOTIFICATION_EVENTS.dup
             rc.add_option! :attribute_namespace, "rails/"
           end
+          c.add_option! :middleware_placement, :end,
+                        match: [:begin, :end, Class]
         end
 
         unless config.respond_to? :opencensus
           config.opencensus = OpenCensus.configure
         end
 
         initializer "opencensus.trace" do |app|
-          app.middleware.insert_before ::Rack::Runtime, RackMiddleware
+          setup_middleware app.middleware
           setup_notifications
         end
 
+        ##
+        # Insert middleware into the middleware stack
+        # @private
+        #
+        def setup_middleware middleware_stack
+          where = OpenCensus::Trace.configure.middleware_placement
+          case where
+          when Class
+            middleware_stack.insert where, RackMiddleware
```

Comment:
should this be `insert_before`?

Reply:
One is aliased to the other. But you're right, it should be explicit.